.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+**Description**
+
+*Include a high-level description of the feature or error here including steps
+of how to recreate it if applicable. Include any benefits, challenges, or
+considerations. This can be short and sweet.*
+
+**Ask**
+
+*Describe the desired behavior and what would deem this issue, bug, or feature
+complete.*
+
+**To Do**
+- [ ] Steps
+- [ ] To
+- [ ] Complete/Fix
+
+**Additional Info**
+
+*Include any images, steps to recreate, notes, emojis, or whatever.*
```

Comment:
This is purely a stylistic recommendation, but I'd consider replacing "or whatever" (which could sound flippant) with "or anything else that seems pertinent."